sale_payment_sheet/readme/DESCRIPTION.md

```diff
@@ -0,0 +1,5 @@
+This module allows salesmen to register payments in a new document
```

Comment:
@victoralmau @sergio-teruel It should be great to explain a little bit the context. As we don't know the 'why'. Thanks

Reply:
@victoralmau @sergio-teruel